src/controllers/references.ts

```diff
@@ -30,7 +31,33 @@ export const getReferencesByType = async (
 ) => {
   try {
     const { type } = req.params;
+    const { populate } = req.query;
     const data = await DefinedReference.find({ type });
+
+    if (type === "roles" && populate) {
```

Comment:
Verify populate as it should be a string in the query

Reply:
populate verified if equal "1" or "true"